src/main/java/pl/byczazagroda/trackexpensesappbackend/model/Wallet.java

```diff
@@ -0,0 +1,36 @@
+package pl.byczazagroda.trackexpensesappbackend.model;
+
+import lombok.*;
+
+import java.time.OffsetDateTime;
+import java.util.UUID;
+import javax.persistence.*;
+
+@Entity
+@Data
+@Table(name = "wallet")
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+public class Wallet {
+
+  @Id
+  @GeneratedValue(strategy = GenerationType.IDENTITY)
+  @EqualsAndHashCode.Include
+  @Column(updatable = false)
+  private Long id;
+
+  @ToString.Include
+  @Column(nullable = false)
+  private UUID walletId;
```

Comment:
I don't get reason, why u implemented second ID here. If you look up to issue you can see following -> 
"The wallet entity should contain:
Id,
wallet name,
wallet creation date.
Wallet name:"